nodejs18.x/full-stack/{{cookiecutter.project_name}}/deploy_frontend.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Get user input for stack name
+read -p "Enter the name of the CloudFormation stack: " stack_name
+
+# Get the API Gateway URL from the stack
+api_gateway_endpoint=$(aws cloudformation describe-stacks --stack-name "$stack_name" --query "Stacks[0].Outputs[?OutputKey=='APIGatewayEndpoint'].OutputValue" --output text)
+
+# Get the CloudFront Distribution ID from the stack
+cloudfront_distribution_id=$(aws cloudformation describe-stacks --stack-name "$stack_name" --query "Stacks[0].Outputs[?OutputKey=='CloudFrontDistributionId'].OutputValue" --output text)
+
+# Get the S3 Bucket Name from the stack
+s3_bucket_name=$(aws cloudformation describe-stacks --stack-name "$stack_name" --query "Stacks[0].Outputs[?OutputKey=='WebS3BucketName'].OutputValue" --output text)
+
+# Output the results
+echo "API Gateway URL: $api_gateway_endpoint"
+echo "CloudFront Distribution ID: $cloudfront_distribution_id"
+echo "S3 Bucket Name: $s3_bucket_name"
+
+# Move to frontend and install
+cd frontend/ && npm install
+
+# Create .env file for building distribtuion with API Gateway Endpoint defined
+touch .env
+
+# Add the API Gateway endpoint to the .env file
+echo "VUE_APP_API_ENDPOINT=$api_gateway_endpoint" >> .env
+
+# Confirm that the endpoint has been added to the .env file
+echo "The API Gateway endpoint has been added to the .env file:"
+cat .env
+
+# Create distribution for deployment
+npm run build && cd dist/
```

Comment:
can we move these statements to another script, so customers can build the forntend, and customers can test it using `sam local start-api`.

Reply:
Also can update [the Project readme](https://github.com/aws/aws-sam-cli-app-templates/blob/fab526298badadf9f879d908d26395fb2e80b3fc/nodejs18.x/full-stack/%7B%7Bcookiecutter.project_name%7D%7D/README.md) to add a section on how customers can test frontend with `sam local start-api`

Reply:
Project readme is updated with steps to run/test frontend code locally.